src/plugins/footnotes/index.ts

```diff
@@ -422,7 +422,7 @@ export default (props: PluginProps) => {
             )
             decorations.push(
               Decoration.widget(
-                targetNode.pos + (footnote ? 1 : 2),
+                targetNode.pos + 1,
```

Comment:
I remember this change was important to make things work properly. so can you please explain why you need to change?

Reply:
I observed that issue LEAN-3294 affected cited footnotes until we implemented the change "targetNode.pos + (footnote ? 1 : 2)." This adjustment resolved the problem. Applying the same change to general notes also fixed the issue, as well as LEAN-3587. Additionally, it turned out that this modification made no significant difference.